.circleci/workflows.yml

```diff
@@ -28,7 +28,7 @@ mainBuildFilters: &mainBuildFilters
       only:
         - develop
         - /^release\/\d+\.\d+\.\d+$/
-        - 'ryanm/fix/issues-with-v8-snapshot-on-develop'
+        - 'tgriesser/spike/_spike'
```

Comment:
Is there a reason we have the _spike here? We're not actually testing the other platforms here.

Reply:
Yeah I had added it temporarily and then later added the `_` so it wasn't running everything on every commit once I verified what I needed